exporter/clickhouselogsexporter/migrations/000005_attribute_rename.down.sql

```diff
@@ -0,0 +1,32 @@
+RENAME TABLE if exists signoz_logs.logs_attribute_keys TO signoz_logs.logs_atrribute_keys;
```

Comment:
This is confusing honestly. I was wondering why you are adding a spelling mistake until I realized this is a .down file.

Reply:
Yea 😅  don't have any other option